src/ds/ds5/ds5-auto-calibration.cpp

```diff
@@ -602,7 +602,7 @@ namespace librealsense
             }
 
             if (host_assistance == host_assistance_type::no_assistance || host_assistance == host_assistance_type::assistance_start)
-                _hw_monitor->send(command{ ds::AUTO_CALIB, focal_length_calib_begin, fl_step_count, fy_scan_range, p4 });
+                _hw_monitor->send(command{ ds::AUTO_CALIB, focal_length_calib_begin, (unsigned int)fl_step_count, (unsigned int)fy_scan_range, p4 });
```

Comment:
Prefer static_cast to C-style casts (preserves const and more)
command uses uint32_t. unsigned int might be 64bit on some platforms.

Relevant to other changes in this file as well.

Reply:
changed unsigned to uint32_t.